torch_xla/_patched_functions.py

```diff
@@ -31,10 +31,10 @@ def clip_grad_norm_(parameters: _tensor_or_tensors,
   parameters = list(filter(lambda p: p.grad is not None, parameters))
   max_norm = float(max_norm)
   norm_type = float(norm_type)
+  if len(parameters) == 0:
+    return torch.tensor(0.)
```

Comment:
why return a cpu tensor here?

Reply:
Good call. Let me change it to xm.xla_device().